core/src/test/kotlin/org/evomaster/core/search/gene/GeneSamplerForTests.kt

```diff
@@ -291,7 +291,7 @@ object GeneSamplerForTests {
                 numberOfOctets = rand.nextInt(1, MAX_NUMBER_OF_OCTETS))
     }
 
-    const val MAX_NUMBER_OF_DIMENSIONS = 5
+    const val MAX_NUMBER_OF_DIMENSIONS = 3 //5
```

Comment:
add comment, stating that larger number can lead to insane quantity of genes, even hundreds of thousands